### DISCUSSION TITLE
X-store 桃園也有

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#740] [General]
X-store是7-11推出的無人商店（桃園大園區也有ㄌ）
我認爲無人商店的好處是可以直接的取得消費者喜好資料
了解客群取向與偏好
藉此分析每月或年營銷與銷售趨勢
或是藉由數據資料研擬新的銷售策略
https://www.bnext.com.tw/article/67024/7-11-x-store-5?

**Comment by HsuChen88** [comment]
除了可以節省人力成本外，對我來說，可以不用看到店員厭世的臉更開心，學校的店員感覺每天心情都很差。

**Reply by wu51000** [reply]
有志一同……
對於社恐的人完全是個福音（？？

**Comment by yjpxi** [comment]
雖然無人商店提高了便利性，但也失去了傳統商店中與顧客互動的個性化服務，有顧客可能會覺得缺乏溫暖和人情味，特別是在需要幫助或諮詢時，無人商店的機械化操作無法提供及時且有效的支持。
